README.md

```diff
@@ -1,18 +1,22 @@
+<br>
+
 <div align="center">
   <picture>
     <source media="(prefers-color-scheme: dark)" srcset="https://github.com/daytonaio/daytona/raw/main/assets/images/Daytona-logotype-white.png">
     <img alt="Daytona logo" src="https://github.com/daytonaio/daytona/raw/main/assets/images/Daytona-logotype-black.png" width="40%">
   </picture>
 </div>
 
-<br><br>
+<br>
 
 <div align="center">
 
+[![Documentation](https://img.shields.io/badge/Documentation-23cc71)](https://www.daytona.io/docs)
```

Comment:
@idagelic would keep the standard of badges like this: https://img.shields.io/badge/Docs-v.0.24-green Ideally you can make the version dynamic to be the same as the version of daytona

Reply:
@ivan-burazin sure, done